### PR TITLE
Patch 1.6.1 Without yaml lint fixes

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
CSM-Operator changes for CSM 1.11.1, without yaml lint fixes. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|  https://github.com/dell/csm/issues/1478 |
| https://github.com/dell/csm/issues/1482 |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x]  Installed csm-operator successfully
- [x]  Unity E2E test passes
- [x] PowerStore E2E tests pass 
- [ ] PowerFlex E2E tests 
- [ ] PowerScale E2E tests
- [ ] Powermax E2E tests

### CODE REVIEW

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      # fsGroupPolicy: Defines if the underlying volume supports changing ownership and permission of the volume before being mounted.
+      # Allowed values: ReadWriteOnceWithFSType, File , None
+      # Default value: ReadWriteOnceWithFSType
+      fSGroupPolicy: "File"
+      # storageCapacity: Helps the scheduler to schedule the pod on a node satisfying the topology constraints, only if the requested capacity is available on the storage array
+      # Allowed values:
+      #   true: enable storage capacity tracking
+      #   false: disable storage capacity tracking
+      storageCapacity: true
+    configVersion: v2.11.0
```
Comment (rishabhatdell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:19]: shouldn't it be 
`configVersion: v2.11.1`

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -4618,9 +4614,9 @@ spec:
                 - name: RELATED_IMAGE_csipowermax-reverseproxy
                   value: docker.io/dellemc/csipowermax-reverseproxy:v2.10.0
                 - name: RELATED_IMAGE_csi-powerstore
-                  value: docker.io/dellemc/csi-powerstore:v2.11.0
+                  value: docker.io/dellemc/csi-powerstore:v2.11.1
```
Comment (rajendraindukuri) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:4617]: Line no 4613: dell-csm-operator version needs to be updated

Comment (rajendraindukuri) [reply bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:4617]: Also CSM Version needs to be updated in Line no: 4584

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -37,9 +37,9 @@ spec:
           name: RELATED_IMAGE_csi-powermax
         - value: docker.io/dellemc/csipowermax-reverseproxy:v2.10.0
           name: RELATED_IMAGE_csipowermax-reverseproxy
-        - value: docker.io/dellemc/csi-powerstore:v2.11.0
+        - value: docker.io/dellemc/csi-powerstore:v2.11.1
```
Comment (rajendraindukuri) [inline config/manager/manager.yaml:40]: Same here.. Please update the version for CSM and dell-csm-operator in line no s 18 and 32 respectively

File: config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -2074,4 +2074,4 @@ spec:
     name: cert-persister
   skips:
   - dell-csm-operator.v1.5.1
```
Comment (rajendraindukuri) [inline config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml:2076]: the version above should be updated to 1.6.0

Comment (rajendraindukuri) [inline config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml:2076]: Here the version should be updated to v1.6.0

File: operatorconfig/driverconfig/powerstore/v2.11.1/driver-config-params.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  name: <DriverDefaultReleaseName>-config-params
+  namespace: <DriverDefaultReleaseNamespace>
+data:
+  driver-config-params.yaml: |
+    CSI_LOG_LEVEL: "debug"
+    CSI_LOG_FORMAT: "JSON"
+    PODMON_CONTROLLER_LOG_LEVEL: "debug"
+    PODMON_CONTROLLER_LOG_FORMAT: "JSON"
+    PODMON_NODE_LOG_LEVEL: "debug"
+    PODMON_NODE_LOG_FORMAT: "JSON"
```
Comment (rajendraindukuri) [inline operatorconfig/driverconfig/powerstore/v2.11.1/driver-config-params.yaml:29]: Add a new line at the end

File: operatorconfig/driverconfig/unity/v2.11.1/csidriver.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+    name: csi-unity.dellemc.com
+spec:
+    attachRequired: true
+    podInfoOnMount: true
+    storageCapacity: true
+    volumeLifecycleModes:
+      - Persistent
+      - Ephemeral
+    fsGroupPolicy: ReadWriteOnceWithFSType
```
Comment (rajendraindukuri) [inline operatorconfig/driverconfig/unity/v2.11.1/csidriver.yaml:12]: Add a new line at the end

File: samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml [hunk 7]
```diff
@@ -0,0 +1,406 @@
+apiVersion: storage.dell.com/v1
```
Comment (rajendraindukuri) [inline samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml:1]: I suggest to remove this file as this should  be updated only after we publish the images to redhat ..only then  we get the SHA ids for the changed images

File: samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml [hunk 8]
```diff
@@ -0,0 +1,216 @@
+#
```
Comment (rajendraindukuri) [inline samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml:1]: Same comment as above for all OCP samples

File: samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml (hunk 7 above)
Comment (suryagupta4) [reply samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml:1]: @rajendraindukuri : The file was added because we had to update the new SHA of SDC 4.5.2.1, this file has the updated SHA

File: samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml (hunk 8 above)
Comment (suryagupta4) [reply samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml:1]: @rajendraindukuri : So all the OCP samples will get added later not now? If yes, we need to make sure to update the latest SHA of SDC in the storage_csm_powerflex_v2110.yaml file whenever it will get added.

Comment (rajendraindukuri) [reply samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml:1]: @suryagupta4 All OCP samples will be updated after publishing to Redhat. If we need to update 1.11 sample, we can directly update here https://github.com/dell/csm-operator/tree/main/samples/ocp/1.6.0 (which is on main branch) if the patch release branch is not carved out of main.

Comment (suryagupta4) [reply samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml:1]: yes, the patch release branch is not carved out of main, the SDC latest SHA needs to be updated in main branch as well. We will raise another PR for main to update that. @rishabhatdell: Please create a PR for updating the SDC SHA against main branch.

For this patch release, we created `ocp/1.6.1` folder inside which all the samples are present which are required for patch release (which are powerstore and unity v2.11.1) and powerflex v2.11.0 was also added since it will have the updated SHA id for SDC. If the samples have to be removed, shall we create the `ocp/1.6.1` and keep the samples in the main branch?

File: samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml (hunk 7 above)
Comment (rishabhatdell) [reply samples/ocp/1.6.1/storage_csm_powerflex_v2110.yaml]: Removed the ocp/.1.6.1 samples as the YAMLs will be updated after the image is published to Red Hat by @rajendraindukuri

File: samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml (hunk 8 above)
Comment (rishabhatdell) [reply samples/ocp/1.6.1/storage_csm_powerstore_v2111.yaml]: Removed the ocp/1.6.1 samples as the YAMLs will be updated after the image is published to Red Hat by @rajendraindukuri

File: operatorconfig/driverconfig/unity/v2.11.1/csidriver.yaml (hunk 6 above)
Comment (ChristianAtDell) [reply operatorconfig/driverconfig/unity/v2.11.1/csidriver.yaml]: Is the ask here to go through every YAML file and ensure that they have a newline at the end? That could take hours and would be easily resolved by a linter, but we're pulling the linter changes out because it was almost fifty thousand additional lines of change.

File: bundle/metadata/annotations.yaml [hunk 9]
```diff
@@ -15,4 +15,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.15-v4.16
+  com.redhat.openshift.versions: "v4.15"
```
Comment (JacobGros) [inline bundle/metadata/annotations.yaml:18]: probably want the range "v4.15-v4.16"

**Review by ChristianAtDell — APPROVED** [review]
There appear to be very few changes that aren't YAML modifications or testfiles. I'm not entirely sure what the contents of this patch are.

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 10]
```diff
@@ -3416,7 +3416,9 @@ spec:
     Drivers\n*\tResiliency - Designed to make Kubernetes applications more resilient
     to \n  node failures\n\nThese modules are available for various storage platforms
     in the Dell portfolio, \nsuch as Dell PowerStore, Dell PowerMax, Dell PowerFlex,
-    Dell PowerScale, and Dell Unity XT.\n\nFor more information, visit [Dell’s Container
+    Dell PowerScale, and Dell Unity XT.\n\n### Supported Openshift Platforms\n\nQualified Openshift versions for CSM Operator v1.6.0: 4.15-4.16
```
Comment (JacobGros) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:3419]: Do we need to update Operator versions here or not since this is a patch?

File: bundle/metadata/annotations.yaml (hunk 9 above)
[the remark this answers is not in the document]
Comment (JacobGros) [reply bundle/metadata/annotations.yaml]: I added the range back, we reviewed the same thing on the older PR and it was agreed we want the range

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml (hunk 10 above)
Comment (JacobGros) [reply bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:3419]: looks like this is the first patch to include this line.
Since it's 1.6.0 was fully qualified, I'll leave it as is

**Review by abhi16394 — APPROVED** [review]
LGTM

File: bundle/metadata/annotations.yaml [hunk 11]
```diff
@@ -15,4 +15,4 @@ annotations:
   operators.operatorframework.io.test.config.v1: tests/scorecard/
 
   # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.15-v4.16
+  com.redhat.openshift.versions: "v4.15-v4.16"
```
Comment (gallacher) [inline bundle/metadata/annotations.yaml:18]: Why is this change required for this version of the csm-operator? Why wasnit it required for 1.6.0?

File: tests/e2e/array-info.sh [hunk 12]
```diff
@@ -39,7 +39,8 @@ export PFLEX_TENANT_PREFIX="tn1"
 export PSCALE_CLUSTER="Isilon-System-Name"
 export PSCALE_USER="username"
 export PSCALE_PASS="password"
-export PSCALE_ENDPOINT="1.1.1.1:8080"
+export PSCALE_ENDPOINT="1.1.1.1"
```
Comment (gallacher) [inline tests/e2e/array-info.sh:42]: Why is this change required for this version of the csm-operator? Why wasnit it required for 1.6.0?

File: tests/e2e/go.mod [hunk 13]
```diff
@@ -34,7 +34,7 @@ require (
 	github.com/evanphx/json-patch/v5 v5.6.0 // indirect
 	github.com/felixge/httpsnoop v1.0.3 // indirect
 	github.com/fsnotify/fsnotify v1.6.0 // indirect
-	github.com/go-logr/logr v1.4.1 // indirect
+	github.com/go-logr/logr v1.4.2 // indirect
```
Comment (gallacher) [inline tests/e2e/go.mod:37]: These libraries should be the same as those released as part of 1.6.0

File: tests/e2e/steps/steps_def.go [hunk 14]
```diff
@@ -53,9 +53,9 @@ var (
...
 	pflexAuthSecretMap     = map[string]string{"REPLACE_USER": "PFLEX_USER", "REPLACE_SYSTEMID": "PFLEX_SYSTEMID", "REPLACE_ENDPOINT": "PFLEX_AUTH_ENDPOINT", "REPLACE_MDM": "PFLEX_MDM"}
-	pscaleSecretMap        = map[string]string{"REPLACE_CLUSTERNAME": "PSCALE_CLUSTER", "REPLACE_USER": "PSCALE_USER", "REPLACE_PASS": "PSCALE_PASS", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT"}
-	pscaleAuthSecretMap    = map[string]string{"REPLACE_CLUSTERNAME": "PSCALE_CLUSTER", "REPLACE_USER": "PSCALE_USER", "REPLACE_PASS": "PSCALE_PASS", "REPLACE_AUTH_ENDPOINT": "PSCALE_AUTH_ENDPOINT", "REPLACE_PORT": "PSCALE_AUTH_PORT", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT"}
-	pscaleAuthSidecarMap   = map[string]string{"REPLACE_CLUSTERNAME": "PSCALE_CLUSTER", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT", "REPLACE_AUTH_ENDPOINT": "PSCALE_AUTH_ENDPOINT", "REPLACE_PORT": "PSCALE_AUTH_PORT"}
+	pscaleSecretMap        = map[string]string{"REPLACE_CLUSTERNAME": "PSCALE_CLUSTER", "REPLACE_USER": "PSCALE_USER", "REPLACE_PASS": "PSCALE_PASS", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT", "REPLACE_PORT": "PSCALE_PORT"}
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:56]: Why are these changes required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 15]
```diff
@@ -67,7 +67,7 @@ var (
...
 
 	// Auth V2
-	pscaleCrMap = map[string]string{"REPLACE_STORAGE_NAME": "PSCALE_STORAGE", "REPLACE_STORAGE_TYPE": "PSCALE_STORAGE", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT", "REPLACE_SYSTEM_ID": "PSCALE_CLUSTER", "REPLACE_VAULT_STORAGE_PATH": "PSCALE_VAULT_STORAGE_PATH", "REPLACE_ROLE_NAME": "PSCALE_ROLE", "REPLACE_QUOTA": "PSCALE_QUOTA", "REPLACE_STORAGE_POOL_PATH": "PSCALE_POOL_V2", "REPLACE_TENANT_NAME": "PSCALE_TENANT", "REPLACE_TENANT_ROLES": "PSCALE_ROLE", "REPLACE_TENANT_VOLUME_PREFIX": "PSCALE_TENANT_PREFIX"}
+	pscaleCrMap = map[string]string{"REPLACE_STORAGE_NAME": "PSCALE_STORAGE", "REPLACE_STORAGE_TYPE": "PSCALE_STORAGE", "REPLACE_ENDPOINT": "PSCALE_ENDPOINT", "REPLACE_SYSTEM_ID": "PSCALE_CLUSTER", "REPLACE_VAULT_STORAGE_PATH": "PSCALE_VAULT_STORAGE_PATH", "REPLACE_ROLE_NAME": "PSCALE_ROLE", "REPLACE_QUOTA": "PSCALE_QUOTA", "REPLACE_STORAGE_POOL_PATH": "PSCALE_POOL_V2", "REPLACE_TENANT_NAME": "PSCALE_TENANT", "REPLACE_TENANT_ROLES": "PSCALE_ROLE", "REPLACE_TENANT_VOLUME_PREFIX": "PSCALE_TENANT_PREFIX", "REPLACE_PORT": "PSCALE_PORT"}
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:70]: Why are these changes required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 16]
```diff
@@ -328,7 +328,7 @@ func (step *Step) deleteCustomResource(res Resource, crNumStr string) error {
 func (step *Step) validateCustomResourceStatus(res Resource, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
 	cr := res.CustomResource[crNum-1]
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:331]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 17]
```diff
@@ -346,13 +346,13 @@ func (step *Step) validateCustomResourceStatus(res Resource, crNumStr string) er
 
 func (step *Step) validateDriverInstalled(res Resource, driverName string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:349]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 18]
```diff
@@ -346,13 +346,13 @@ func (step *Step) validateCustomResourceStatus(res Resource, crNumStr string) er
...
 	return checkAllRunningPods(context.TODO(), res.CustomResource[crNum-1].Namespace, step.clientSet)
 }
 
 func (step *Step) validateDriverNotInstalled(res Resource, driverName string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:355]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 19]
```diff
@@ -379,7 +379,7 @@ func (step *Step) removeNodeLabel(res Resource, label string) error {
 func (step *Step) validateModuleInstalled(res Resource, module string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
 	cr := res.CustomResource[crNum-1]
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:382]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 20]
```diff
@@ -424,7 +424,7 @@ func (step *Step) validateModuleInstalled(res Resource, module string, crNumStr
 func (step *Step) validateModuleNotInstalled(res Resource, module string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
 	cr := res.CustomResource[crNum-1]
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:427]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 21]
```diff
@@ -889,7 +889,7 @@ func (step *Step) runCustomTest(res Resource) error {
 func (step *Step) enableModule(res Resource, module string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
 	cr := res.CustomResource[crNum-1]
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:892]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/steps/steps_def.go [hunk 22]
```diff
@@ -1303,6 +1303,16 @@ func (step *Step) AuthorizationV1Resources(storageType, driver, port, proxyHost,
 	// get env variables
 	if os.Getenv(endpointvar) != "" {
 		endpoint = os.Getenv(endpointvar)
+
```
Comment (gallacher) [inline tests/e2e/steps/steps_def.go:1306]: Why is this change required for 1.6.1 and not 1.6.0?

File: tests/e2e/testfiles/powerscale-templates/karavi-authorization-config.json [hunk 23]
```diff
@@ -1,8 +1,8 @@
 [{
     "username":"-",
     "password":"-",
-    "intendedEndpoint":"https://REPLACE_ENDPOINT",
-    "endpoint":"https://REPLACE_AUTH_ENDPOINT:REPLACE_PORT",
+    "intendedEndpoint":"https://REPLACE_ENDPOINT:REPLACE_PORT",
```
Comment (gallacher) [inline tests/e2e/testfiles/powerscale-templates/karavi-authorization-config.json:4]: Why is this change required for 1.6.1 and not 1.6.0?

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 24]
```diff
@@ -4779,5 +4775,5 @@ spec:
   - image: docker.io/dellemc/connectivity-cert-persister-k8s:0.11.0
     name: cert-persister
   skips:
-  - dell-csm-operator.v1.5.1
+  - dell-csm-operator.v1.6.0
   version: 1.6.0
```
Comment (rajendraindukuri) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:4779]: Version should be 1.6.1

File: tests/e2e/array-info.sh (hunk 12 above)
Comment (ChristianAtDell) [reply tests/e2e/array-info.sh:42]: Technically speaking, this change is not required for it to work, because the field is replaced with whatever the end-user actually wants their array credentials to be: it's just a placeholder value. That placeholder value SHOULD only be an IP.

However, leaving it as 1.1.1.1:8080 communicates the wrong information to the user, that they should replace the value as `<IP>:<PORT>` when they really should just replace it as `<IP>`. Leaving it as it was could lead someone to put `<IP>:<PORT>` when running it themselves, and they will meet issues where it tries to deploy to `<IP>:<PORT>:<PORT>` which won't work. Removing the port from this line resolves that confusion/problem.

File: tests/e2e/steps/steps_def.go (hunk 17 above)
Comment (ChristianAtDell) [reply tests/e2e/steps/steps_def.go:349]: I don't believe this change is required for anything, it just reduces the amount of unnecessary delay in the testing substantially to accelerate the test runs.

File: tests/e2e/testfiles/scenarios.yaml [hunk 25]
```diff
@@ -478,7 +482,6 @@
     - "authorization"
     - "powerscale"
     - "observability"
-    # - "sanity"
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/scenarios.yaml:481]: I understand removing a commented-out tag, but should this tag actually be existent? What are the guidelines for a sanity test, and should this be included among them?

File: tests/e2e/testfiles/scenarios.yaml [hunk 26]
```diff
@@ -547,8 +551,10 @@
     - "Delete custom resource [1]"
     - "Restore template [testfiles/powerscale-templates/powerscale-secret-template.yaml] for [pscale]"
     - "Restore template [testfiles/powerscale-templates/karavi-authorization-config.json] for [pscaleAuthSidecar]"
+    - "Restore template [testfiles/powerscale-templates/powerscale-auth-secret-template.yaml] for [pscaleAuth]"
+    - "Restore template [testfiles/authorization-templates/rootCertificate.pem] for [authSidecarCert]"
 
-- scenario: "Install PowerScale Driver(With Authorization V1 and Observability), Disable Authorization module, Disable Observability module"
+- scenario: "Install PowerScale Driver(With Authorization V1 and Observability), Disable Observability module, Disable Authorization module"
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/scenarios.yaml:557]: Do we document anywhere that observability must be removed before authorization? @falfaroc

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml (hunk 1 above)
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:19]: Should this be 2.11.1?

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml [hunk 27]
```diff
@@ -0,0 +1,225 @@
...
+    replicas: 1
+    dnsPolicy: ClusterFirstWithHostNet
+    forceUpdate: false
+    forceRemoveDriver: true
+    common:
+      image: "dellemc/csi-vxflexos:nightly"
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:25]: Should our tests pull nightly images, or should they pull specific versions?

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml [hunk 28]
```diff
@@ -0,0 +1,225 @@
...
+      imagePullPolicy: IfNotPresent
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:26]: This, paired with `nightly` tag especially, worries me on testing: it would not update the image if the user has any `nightly` image, correct? That image could be days, weeks, months old, and still fit the `nightly` tag.

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml [hunk 29]
```diff
@@ -0,0 +1,225 @@
...
+    # Replication: allows to configure replication
+    # Replication CRDs must be installed before installing driver
+    - name: replication
+      # enabled: Enable/Disable replication feature
+      # Allowed values:
+      #   true: enable replication feature(install dell-csi-replicator sidecar)
+      #   false: disable replication feature(do not install dell-csi-replicator sidecar)
+      # Default value: false
+      enabled: true
+      configVersion: v1.9.0
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:175]: Is this an appropriate configVersion for this release?

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml [hunk 30]
```diff
@@ -0,0 +1,225 @@
...
+      components:
+        - name: dell-csi-replicator
+          # image: Image to use for dell-csi-replicator. This shouldn't be changed
+          # Allowed values: string
+          # Default value: None
+          image: dellemc/dell-csi-replicator:nightly
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:181]: `nightly` tag appropriate for this? It might save some time on updating the YAMLs from release to release, but it is less transparent/predictable on the image used.

File: tests/shared/common.go [hunk 31]
```diff
@@ -35,8 +35,8 @@ const (
 	JumpDowngradeConfigVersion string = "v2.10.0"
 	OldConfigVersion           string = "v2.2.0"
 	BadConfigVersion           string = "v0"
-	PStoreConfigVersion        string = "v2.11.0"
-	UnityConfigVersion         string = "v2.11.0"
+	PStoreConfigVersion        string = "v2.11.1"
+	UnityConfigVersion         string = "v2.11.1"
 	PScaleConfigVersion        string = "v2.11.0"
 	PmaxConfigVersion          string = "v2.11.0"
 	AuthServerConfigVersion    string = "v1.11.0"
```
Comment (ChristianAtDell) [inline tests/shared/common.go:42]: Why did only some of these get bumped to 2.11.1?

File: samples/storage_csm_powerstore_v2111.yaml [hunk 32]
```diff
@@ -0,0 +1,216 @@
...
+      #   effect: "NoSchedule"
+  modules:
+    - name: resiliency
+      # enabled: Enable/Disable Resiliency feature
+      # Allowed values:
+      #   true: enable Resiliency feature(deploy podmon sidecar)
+      #   false: disable Resiliency feature(do not deploy podmon sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.10.0
```
Comment (ChristianAtDell) [inline samples/storage_csm_powerstore_v2111.yaml:180]: Is this the right configVersion?

File: tests/e2e/testfiles/storage_csm_powermax_reverseproxy_authorization.yaml [hunk 33]
```diff
@@ -270,11 +270,11 @@ spec:
       configVersion: v1.11.0
```
Comment (ChristianAtDell) [inline tests/e2e/testfiles/storage_csm_powermax_reverseproxy_authorization.yaml:270]: Should this be bumped to 1.11.1?

File: tests/e2e/testfiles/scenarios.yaml (hunk 25 above)
Comment (falfaroc) [reply tests/e2e/testfiles/scenarios.yaml:481]: That is a great question and the quick answer is, not to my knowledge. This is something that can be improved in future releases.

File: tests/e2e/testfiles/scenarios.yaml (hunk 26 above)
Comment (falfaroc) [reply tests/e2e/testfiles/scenarios.yaml:557]: I believe that it would work but reconfiguration would be needed of the observability module which the tests currently don't support. To limit the changes, the order of disabling was changed.

We can either, leave this like this and update the tests in the future or invest more time into the tests for this release.

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml (hunk 1 above)
Comment (ChristianAtDell) [reply tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:19]: No. We are not releasing a PowerFlex 2.11.1, so it stays 2.11.0.

File: tests/e2e/testfiles/storage_csm_powerflex_replica.yaml (hunk 27 above)
Comment (ChristianAtDell) [reply tests/e2e/testfiles/storage_csm_powerflex_replica.yaml:25]: Confirmed with @falfaroc that they should match the ConfigVersion. Updating.

File: tests/e2e/testfiles/scenarios.yaml (hunk 26 above)
Comment (ChristianAtDell) [reply tests/e2e/testfiles/scenarios.yaml:557]: Leaving it as-is seems fine for now.

**Review by ChristianAtDell — APPROVED** [review]
